app/backend/src/couchers/models.py

```diff
@@ -653,3 +653,19 @@ class InitiatedUpload(Base):
     @hybrid_property
     def is_valid(self):
         return (self.created <= func.now()) & (self.expiry >= func.now())
+
+
+class Node(Base):
+    """
+    Node, i.e. geographical subdivision of the world
+    """
+
+    __tablename__ = "nodes"
+
+    id = Column(Integer, primary_key=True)
+
+    name = Column(String, nullable=False)
+    parent_node = Column(Integer, ForeignKey("nodes.id"))
```

Comment:
Should be nullable?

Reply:
The top (world-wide) node does not have a parent. An alternative is to have it point to itself. Or we use zero. Ok, zero it will be.

Reply:
Db calls might be easier with NULL for the world node